packages/dialtone-vue3/recipes/comboboxes/combobox_multi_select/combobox_multi_select.vue

```diff
@@ -1,3 +1,4 @@
+<!-- eslint-disable vue/no-static-inline-styles -->
```

Comment:
why is this line needed? The same applies for Vue 2 version

Reply:
because we're setting `:style` directly on the component which is generally not something we want to do, but in this case it makes sense.